frontend/src/app/pages/home-page/home-page.component.html

```diff
@@ -1,20 +1,28 @@
-<div class="menu">
-    <div class="button-row">
-        <button mat-fab routerLink="{{ user.settings.skipTutorial ? '/labeling/choose-task' : '/labeling/tutorial' }}">
-            <mat-icon>label_outline</mat-icon>
-            <div>Labeling</div>
-        </button>
-        <button mat-fab routerLink="/upload" *ngIf="user.role !== 'volunteer'">
-            <mat-icon>add</mat-icon>
-            <div>Upload new Scans</div>
-        </button>
-        <button mat-fab (click)="indicateValidationPageIsUnavailable()" *ngIf="user.role !== 'volunteer'">
-            <mat-icon>assignment_turned_in</mat-icon>
-            <div>Validation</div>
-        </button>
-        <button mat-fab routerLink="/settings">
-            <mat-icon>settings</mat-icon>
-            <div>Settings</div>
-        </button>
-    </div>
-</div>
+<app-nav-bar>
+    <button mat-raised-button color="primary" id="dropdown-button" class="account-button" (blur)="toggleDropdown()" (click)="toggleDropdown()" [ngClass]="{ 'active' : isDropdownActive}">
+        <mat-icon class="account-button__icon mr-2">person</mat-icon>
+        <span class="account-button__name">{{user.firstName + ' ' + user.lastName}}</span>
+        <span *ngIf="isDropdownActive" [@displayDropdown]="isDropdownActive" class="account-button__options">

```

Comment:
I'm having issues with these dropdown elements:
- on Chromium (ver 70.0), these items are highlighted when hovered over (cursor also changes), but clicking on them does nothing,
- on Firefox (ver 63), items  are not highlighted when hovered over, mouse cursor also doesn't change

Reply:
Fixed that!